package.json

```
@@ -45,7 +45,8 @@
     "android": "npm run android:emulator & npm run android:build",
     "spellcheck": "mdspell -r --en-gb --ignore-acronyms --ignore-numbers --no-suggestions '*.md' '**/*.md' '!**/node_modules/**/*.md' '!ios/Pods/**/*.md'",
     "spellcheck:interactive": "mdspell --en-gb --ignore-acronyms --ignore-numbers '*.md' '**/*.md' '!**/node_modules/**/*.md' '!ios/Pods/**/*.md'",
-    "start": "react-native start"
+    "start": "react-native start",
+    "fix:package-lock": "git status -s | grep package-lock.json | awk '{print $2}' | xargs replace --silent 'http://' 'https://'"
```

We should make this run this in `postinstall` - but that won't work on CI as Travis doesn't actually run within the git repo.
So we would need something like this to make Travis happy with running as a postinstall. This command is becomming an absolute unit 😆

```suggestion
    "fix:package-lock": "(git rev-parse --is-inside-work-tree && git status -s | grep package-lock.json | awk '{print $2}' | xargs replace --silent 'http://' 'https://') || true",
```

---

Alternatively we make it a pre-commit hook, or we just skip the git check, so then we don't need to worry about CI 🤷‍♂️